addons/mail/static/src/core/common/chat_hub_model.js

```diff
@@ -4,6 +4,7 @@ import { Record } from "./record";
 import { Deferred, Mutex } from "@web/core/utils/concurrency";
 
 export const CHAT_HUB_KEY = "mail.ChatHub";
+const CHAT_HUB_COMPACT_LS = "mail.user_setting.chathub_compact";
```

Comment:
nitpicking - I am surprised that we used `mail.user_setting` for the prefix rather than `mail.user_settings`, maybe with `-s` is a better option?

Reply:
In a sense this is a single user setting. But we could also say this is in the category of "user settings".
I kept `user_setting` for keeping same name, especially when used in stable.